redisbeat/scheduler.py

```diff
@@ -112,11 +111,6 @@ def remove(self, task_key):
             return False
 
     def tick(self):
-        if not self.rdb.exists(self.key):
```

Comment:
It seems not a good idea to remove these codes.
For celery restart, these code would load old task from local persistent backup.
I suggest to solve the promblem in `delete` function, would you mind to do in this way?

Reply:
No, `delete` function works fine. Removed code lines returns already deleted tasks.
Maybe it's good idea to move this lines to __init__

Reply:
ok, you are right.